src/Commands/Handler/Internal/WorldTransferHandler.cs

```diff
@@ -30,6 +33,15 @@ protected override void Handle(WorldTransferCommand command)
 
                     SaveHelpers.LoadLevel(command.World);
 
+                    Singleton<SimulationManager>.instance.m_ThreadingWrapper.QueueMainThread(() =>
+                    {
+                        ClientJoinPanel clientJoinPanel = UIView.GetAView().FindUIComponent<ClientJoinPanel>("MPClientJoinPanel");
+                        if (clientJoinPanel != null)
+                        {
+                            clientJoinPanel.HidePanel(true);
```

Comment:
Why hide the panel here when it also gets hidden in LoadingExtension.cs#36 ? Also why not through UnblockGame?

Reply:
Used UnblockGame and removed unblock in LoadingExtension, because game is unblocked when loading of the save game starts for the joining client